profiles/seccomp/default_linux.go

```diff
@@ -552,6 +552,762 @@ func DefaultProfile() *Seccomp {
+			},
+		},
+		{
+			Names: []string{
+				"unshare",
+			},
+			Action: specs.ActAllow,
+			Args: []*specs.LinuxSeccompArg{
+				{
+					Index:    0,
+					Value:    unix.CLONE_NEWNS | unix.CLONE_NEWUTS | unix.CLONE_NEWIPC | unix.CLONE_NEWUSER | unix.CLONE_NEWPID | unix.CLONE_NEWNET | unix.CLONE_NEWCGROUP,
+					ValueTwo: 0,
+					Op:       specs.OpMaskedEqual,
+				},
+			},
+			Excludes: Filter{
+				MinKernel: &KernelVersion{4, 18},
+				Caps:   []string{"CAP_SYS_ADMIN"},
+			},
+		},
+		{
+			Names: []string{
+				"reboot",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_BOOT"},
+			},
+		},
+		{
+			Names: []string{
+				"chroot",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_CHROOT"},
+			},
+		},
+		{
+			Names: []string{
+				"delete_module",
+				"init_module",
+				"finit_module",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_MODULE"},
+			},
+		},
+		{
+			Names: []string{
+				"acct",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_PACCT"},
+			},
+		},
+		{
+			Names: []string{
+				"kcmp",
+				"pidfd_getfd",
+				"process_madvise",
+				"process_vm_readv",
+				"process_vm_writev",
+				"ptrace",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_PTRACE"},
+			},
+		},
+		{
+			Names: []string{
+				"iopl",
+				"ioperm",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_RAWIO"},
+			},
+		},
+		{
+			Names: []string{
+				"settimeofday",
+				"stime",
+				"clock_settime",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_TIME"},
+			},
+		},
+		{
+			Names: []string{
+				"vhangup",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_TTY_CONFIG"},
+			},
+		},
+		{
+			Names: []string{
+				"get_mempolicy",
+				"mbind",
+				"set_mempolicy",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYS_NICE"},
+			},
+		},
+		{
+			Names: []string{
+				"syslog",
+			},
+			Action: specs.ActAllow,
+			Args:   []*specs.LinuxSeccompArg{},
+			Includes: Filter{
+				Caps: []string{"CAP_SYSLOG"},
+			},
+		},
+	}
+
+	return &Seccomp{
+		DefaultAction: specs.ActErrno,
+		ArchMap:       arches(),
+		Syscalls:      syscalls,
+	}
+}
+
+// DefaultProfileWithoutUserNamespaces defines the allowed syscalls for the default seccomp profile,
+// except that the unprivileged creation of user namespaces is disallowed.
+func DefaultProfileWithoutUserNamespaces() *Seccomp {
```

Comment:
It may be reasonable to add some logic to `DefaultProfile` that would allow it to generate both profiles without all of the code duplication you see here. For this pull request, I was going with the safest simplest thing.